src/content/subgames.cpp

```diff
@@ -288,8 +306,27 @@ std::vector<WorldSpec> getAvailableWorlds()
 	return worlds;
 }
 
-bool loadGameConfAndInitWorld(const std::string &path, const SubgameSpec &gamespec)
+bool loadGameConfAndInitWorld(const std::string &path, const std::string &name,
+		const SubgameSpec &gamespec, bool mustCreateNew, std::string *error)
 {
+	std::string final_path = path;
+
+	// If we're creating a new world, ensure that the path isn't already taken
+	if (mustCreateNew) {
```

Comment:
Low priority: According to the guidelines, local variables and arguments should use underscores.

Suggestion:
```suggestion
	if (create_new_world) {
```